backend/src/test/java/corea/room/repository/RoomRepositoryTest.java

```diff
@@ -1,82 +1,80 @@
 package corea.room.repository;
 
-import corea.DataInitializer;
+import corea.fixture.MemberFixture;
+import corea.fixture.RoomFixture;
+import corea.member.domain.Member;
+import corea.member.repository.MemberRepository;
 import corea.room.domain.Room;
 import corea.room.domain.RoomClassification;
 import corea.room.domain.RoomStatus;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.CsvSource;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.context.annotation.Import;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageRequest;
-import org.springframework.test.context.ActiveProfiles;
 
 import java.time.LocalDateTime;
-import java.util.ArrayList;
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.SoftAssertions.assertSoftly;
 
-@SpringBootTest
-@Import(DataInitializer.class)
-@ActiveProfiles("test")
+@DataJpaTest
 class RoomRepositoryTest {
 
     @Autowired
     private RoomRepository roomRepository;
 
-    @ParameterizedTest
-    @CsvSource(value = {"ANDROID, 2", "FRONTEND, 1", "BACKEND, 3"})
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Test
     @DisplayName("자신이 참여하지 않고, 계속 모집 중인 방들을 모집 마감일이 임박한 순으로 조회할 수 있다.")
-    void findAllByMemberAndClassificationAndStatus(RoomClassification classification, int expectedSize) {
-        Page<Room> rooms = roomRepository.findAllByMemberAndClassificationAndStatus(1, classification, RoomStatus.OPEN, PageRequest.of(0, 8));
+    void findAllByMemberAndClassificationAndStatus() {
+        Member pororo = memberRepository.save(MemberFixture.MEMBER_PORORO());
+        Member joysun = memberRepository.save(MemberFixture.MEMBER_YOUNGSU());
```

Comment:
```suggestion
        Member joyson = memberRepository.save(MemberFixture.MEMBER_YOUNGSU());
```
이하 이름에는 따로 코멘트 달지 않겠습니다!

Reply:
ㅋㅋㅋㅋㅋㅋㅋ 그게 그거 아닌가유?